backend/core/views.py

```
@@ -817,6 +817,11 @@ class VulnerabilityViewSet(BaseModelViewSet):
     def status(self, request):
         return Response(dict(Vulnerability.Status.choices))
 
+    @method_decorator(cache_page(60 * LONG_CACHE_TTL))
+    @action(detail=False, name="Get severity choices")
+    def severity(self, request):
+        return Response(dict(Severity.choices))
```

Ruff (0.8.2)

`Severity` may be undefined, or defined from star imports

(F405)